sonar-php-plugin/src/test/java/org/sonar/plugins/php/MockUtils.java

```diff
@@ -30,5 +33,13 @@
 
   private MockUtils() {
   }
+  
+  public static DefaultFileSystem getDefaultFileSystem() {
```

Comment:
@vilchik-elena Agreed, I had the same feeling. I wanted to avoid renaming a class. Will do.